modules/settings/settings.php

```diff
@@ -208,33 +201,32 @@ public function settings_validate( $new_options ) {
 	 * Validation and sanitization on the settings field
 	 * This method is called automatically/ doesn't need to be registered anywhere
 	 */
-	public function helper_settings_validate_and_save() {
-
+	public static function helper_settings_validate_and_save(): bool {
 		if ( ! isset( $_POST['action'], $_POST['_wpnonce'], $_POST['option_page'], $_POST['_wp_http_referer'], $_POST['submit'] ) || ! is_admin() ) {
 			return false;
 		}
 
 		if ( 'update' != $_POST['action']
```

Comment:
We can add some `===`/`!==` in this function for all of the string comparisons.

Reply:
Thought I fixed that everywhere, but did not looks like.